gsy_framework/sim_results/all_results.py

```diff
@@ -28,7 +34,9 @@ def __init__(self, should_export_plots: bool = False, is_scm: bool = False):
             "cumulative_bills": CumulativeBills(),
             "cumulative_grid_trades": CumulativeGridTrades(),
             "device_statistics": DeviceStatistics(should_export_plots),
+            "forward_device_statistics": ForwardMarketDeviceStatistics(should_export_plots),
             "trade_profile": EnergyTradeProfile(should_export_plots),
+            "forward_trade_profile": ForwardMarketEnergyTradeProfile(should_export_plots),
```

Comment:
If we would like to keep these new keys, then probably we should also include this change in [results_schema](https://github.com/gridsingularity/gsy-framework/blob/072e780a8ac46dc8c9ac516179a337136d577099/gsy_framework/schemas.py#L202) used for validation.